_conferences/big-android-bbq-amsterdam-2015.md

```diff
@@ -0,0 +1,8 @@
+---
+name: "Big Android BBQ Amsterdam"
```

Comment:
If you add another commit with the change and push it to your repo it'l update this pull request.